src/components/conditions/ConditionsPage.jsx

```diff
@@ -7,19 +7,14 @@ export default class ItemsPage extends React.Component {
     }
 
     render() {
-        if (!this.props.data) return "Loading...";
+        if (!this.props.data) return "";

```

Comment:
The quicker modification.
It's redundant if apply the condition to all these pages inside `Main.jsx`.